### PR TITLE
Add new book

### PR DESCRIPTION
**Pull request by LeTatTung**

### CODE REVIEW

File: app/controllers/admin/books_controller.rb [hunk 1]
```diff
@@ -2,12 +2,27 @@ class Admin::BooksController < Admin::AdminController
   before_action :find_book, only: [:show]
 
   def index
-    @books = Book.all
+    @books = Book.paginate page: params[:page], per_page: Settings.per_page
+  end
+
+  def new
+    @book = Book.new
+    @categories = Category.all
```
Comment (nguyenhuutien) [inline app/controllers/admin/books_controller.rb]: Thêm
@authors = Author.all
@pulishers = Publisher.all

File: app/controllers/admin/books_controller.rb [hunk 2]
```diff
@@ -16,4 +31,9 @@ def find_book
       redirect_to root_path
     end
   end
+
+  def book_params
+    params.require(:book).permit :title, :image, :pages, :price, :quantity,
```
Comment (nguyenhuutien) [inline app/controllers/admin/books_controller.rb]: params.require(:book).permit :title, :image, :pages, :price, :quantity,
 :publish_date, :category_id, :author_id, :publisher_id

File: app/assets/javascripts/application.js [hunk 3]
```diff
@@ -5,13 +5,21 @@
...
 //
 //= require jquery
 //= require jquery_ujs
-//= require turbolinks
 //= require bootstrap
+//= require turbolinks
 //= require_tree .
+
+$(document).ready(function(){
+  $(".datepicker").datepicker({
```
Comment (dat-hedspi) [inline app/assets/javascripts/application.js]: trong js dùng single quote

File: app/controllers/admin/books_controller.rb [hunk 4]
```diff
@@ -1,13 +1,34 @@
...
   end
 
   def show
   end
 
+  def create
+    @book = Book.new book_params
+    if @book.save
+      params[:book][:author_ids].each do |author_id|
+        @book.book_authors.create!(author_id: author_id) unless author_id.blank?
```
Comment (dat-hedspi) [inline app/controllers/admin/books_controller.rb]: thử bỏ () đi xem nào :v

**Comment by yeunuthongminh** [comment]
OK

**Pull request merged**